### PR TITLE
Update data.rst

### PR DESCRIPTION
**Pull request by tonyastrofig**
Created SPAN-Ai data table in data.rst from PDMP. This addresses issue #2.

### CODE REVIEW

**Review by ehsteve — APPROVED** [review]
Looks good to me. For future PRs we should include a changelog but this PR just catches up this repo with the other instrument repos so it is not needed here.

**Comment by ehsteve** [comment]
I updated your description to link it to the issue that this solves.

**Comment by tonyastrofig** [comment]
OK thanks Steve!  I'll ping Roberto.

File: docs/user-guide/data.rst [hunk 1]
```diff
@@ -10,13 +10,44 @@ Overview
...
-| Level    | Product                               | Description                           |      
-+==========+=======================================+=======================================+
-| 1        | Vector magnetic field from 3 sensors. | CCSDS, each packet is 4-sec long      |
-|          |                                       | at 10 Hz rate, 3-axis magnetic        |
-|          |                                       | field components for all three sensors|
-+----------+---------------------------------------+---------------------------------------+
++----------+---------------------------------------+---------------------------------------------------------------------------------------+
+| Level    | Product                               | Description                                                                           |      
++==========+=======================================+=======================================================================================+
+| 1        | H+ Counts                             | 32 Energies x 8 Deflections (256 HV step) x 16 Anodes every 0.87s (may be compressed) |
```
Comment (robertolivi) [inline docs/user-guide/data.rst:16]: Technically, there are 1024 steps in the sweep. We use microstepping, where every 4 steps are summed together, resulting in 256 steps for each anode. The data product is an array of size 4096 bytes.

**Comment by robertolivi** [comment]
The products presented are H+, He++, and O+, at a cadence of 0.87s. These were preliminary product types / cadences that may change as I start working more closely with the IDS teams. When is this document due? How much time do I have to adjust these values? Also, we have housekeeping and diagnostic products. Should they be included here as well?

**Review by robertolivi — APPROVED** [review]
This is a good first step and the information looks mostly correct. As mentioned in the other comment section, these values might slightly change as I start discussing the data products with the IDS teams more carefully.

**Comment by ehsteve** [comment]
@robertolivi we can always make changes to our documentation so if you think it is good enough for now I think we are good to go. We can discuss housekeeping products later. Not sure how relevant they are to every day science users and whether they should be here.

**Pull request merged**